app/components/admin/budgets/duration_component.rb

```diff
@@ -24,6 +24,6 @@ def formatted_end_date
     end
 
     def formatted_date(time)
-      time_tag(time, format: :datetime)
+      time_tag(time, format: :duration)
```

Comment:
The name `duration` is a bit strange because it could be anything :thinking:. "3 days" is a duration, and so is "from January the 27th to February the 13th".

Any ideas for a different name?